src/IO/lbsLoadReaderVT.py

```diff
@@ -140,14 +140,19 @@ def read(self, node_id, time_step=-1, comm=False):
                 n_entries = len(row)
 
                 # Handle three-entry case that corresponds to an object load
-                if n_entries == 3:
+                if '[' in row[4]:
                     # Parsing the three-entry case, thus this format:
-                    #   <time_step/phase>, <object-id>, <time>
+                    #   <time_step/phase>, <object-id>, <time>, <#-subphases> '[' 
+                    #   [<subphase-time-1>] ... [<subphase-time-N>] ']'
                     # Converting these into integers and floats before using them or
                     # inserting the values in the dictionary
                     try:
                         phase, o_id = map(int, row[:2])
                         time = float(row[2])
+                        Nsubphases = int(row[3])
+                        # Trim first and last brackets
+                        row[4] = float(row[4][1:])
```

Comment:
Is this leaving all the subphase times between the first and last as unparsed text?

Reply:
Yes, that would appear to be the case. Given there is a number of sub phases there can be an additional loop that converts the strings to their respective types (floats I would assume).

Reply:
I think you could do the same as seen a couple lines above, with `map(float, row[first:last])`. Given that, it would be nice if trimming out the brackets were applied separately from parsing the values.

I.e.
```
row[4] = row[4][1:]
row[-1] = row[-1][:-1]
row[4:] = map(float, row[4:])
```

Reply:
Actually, better to not modify the row any more, and assign a separate variable

```
subphase_times = row[4:]
assert len(subphase_times) == Nsubphases
subphase_times[0] = subphase_times[0][1:]
subphase_times[1] = subphase_times[-1][:-1]
subphase_times = map(float, subphase_times)
```